kolibri/content/api.py

```diff
@@ -29,6 +30,7 @@ def title_description_filter(self, queryset, value):
         if exact_match:
             return exact_match
         # if no exact match, search for non-adjacent match
+
```

Comment:
Yes, it fixed a bug that happens only in Python 3, where `reduce` is only available in functools.